taxonomy/models.py

```diff
@@ -164,7 +164,6 @@ class ProgramSkill(TimeStampedModel):
     """
 
     program_uuid = models.UUIDField(
-        primary_key=True,
```

Comment:
Why this was removed?

Reply:
A program uuid can have more than one skill. It was violating when more than one skill was added against the same uuid. So we removed this constraint and now we're using the autogenerated primary key aka `id`.